### ISSUE TITLE
Queue Time Length Bug [Latest Version]

### ISSUE THREAD

**Issue by RonWilliam547** [fast4x/RiMusic#1558]
### Steps to reproduce the bug

1) Start playing a song on RiMusic
2) Minimize the player so it's tucked away at the bottom
Swipe up (home button) to get out of the app (not actually closing it)
3) Start playing media from another source (Tested using Spotify)
4) Now return back to RiMusic

P.S.
Thank you to the entire team for creating an amazing music app. I find myself wanting to use this app a lot of times even though I have a premium subscription to Spotify.

### Expected behavior

Correct queue length time should be displayed even after playing media from another source

### Actual behavior

The queue length time will show incorrectly. For me it showed 3 hours and some minutes for the first try and for the second try it showed 2 hours and some minutes. Not sure what causes it to report different time lengths in this case.

The song remaining time hasn't had this bug at least from my testing.

### Screenshots/Screen recordings

![Bugged](https://github.com/fast4x/RiMusic/assets/92063334/74146bbd-9a21-4b50-b854-1c7a80f908c8)
![Expected](https://github.com/fast4x/RiMusic/assets/92063334/0524f7c8-3c6d-4b8d-8f43-1cf3a90d1806)

### RiMusic version

0.6.30

### Android version

Android 14

**Comment by RonWilliam547** [comment]
Yes, you're correct. Apparently the queue is somehow getting messed up when performing those steps then because I didn't add any songs to the queue. I just started playing a song, performed those steps, and that's how it got to 2 hours and 30 minutes.

**Comment by RonWilliam547** [comment]
Oh okay. I didn't see it reported before but I must have missed it. It's not really a big issue for me personally but I can see how it could be for other people especially if they use the queue feature. I haven't really used the queue feature on here yet but I'll definitely get more familiar with the app. Hopefully it gets fixed soon. 😁

**Comment by fast4x** [comment]
When you start a song, queue is populated with certain numbers of songs...

**Comment by fast4x** [comment]
interesting
